src/views/VendorOverview.vue

```diff
@@ -1,66 +1,199 @@
 <template>
-  <div class="vendor-overview container mx-auto mb-8 p-24 space-y-40 pb-3">
-    <div class="flex flex-col items-center space-y-8">
-      <img class="w-30 inset-12" alt="Augustin logo" :src="apiUrl + 'img/logo.png'" />
-      <h2 class="text-3xl font-bold">Dein QR-Code</h2>
-      <img class="w-30 inset-12" alt="vendor qr code" :src="qrcode" />
-    </div>
-    <br />
-    <div class="my-16 container mt-8 pt-8 space-y-4">
-      <h2 class="text-3xl font-bold">Dein Guthaben</h2>
-      <div class="vendor-credit text-6xl font-bold">{{ credit }}€</div>
-      <div class="vendor-number">Ausweisnummer: {{ idNumber }}</div>
-      <div class="vendor-name">Name: Doris</div>
-      <button class="bg-green-500 rounded-full py-3 px-6 text-white font-bold">Logout</button>
-    </div>
-  </div>
+  <component :is="$route.meta.layout || 'div'">
+    <template #main>
+      <!--Main template-->
+      <div className="vendor-overview container mb-8 space-y-2" v-if="authenticated">
+        <div className="flex flex-col items-center space-y-4" v-if="!failure">
+          <h1 className="text-3xl font-bold">{{ $t('yourData') }}</h1>
+          <div class="grid grid-cols-2 place-content-between text-2xl">
+            <strong>{{ $t('menuCredits') }}: </strong>
+            <span class="font-bold" v-if="vendorMe?.Balance !== undefined"
+              >{{
+                vendorMe?.Balance !== undefined
+                  ? (vendorMe?.Balance / 100).toFixed(2)
+                  : 'N/A'
+              }}
+              €</span
+            >
+          </div>
+          <div class="grid grid-cols-2 place-content-between">
+            <strong>{{ $t('lastPayout') }}: </strong>{{ vendorMe?.LastPayout }}
+          </div>
+          <div class="grid grid-cols-2 place-content-between">
+            <strong>{{ $t('lastTransactions') }}: </strong>
+          </div>
+
+          <!--Liste zum scrollen Anfang-->
+          <div class="h-5/6 pb-3">
+            <ul
+              class="list-image-none overflow-y-auto w-full h-full border-4 border-gray-200 rounded-3xl"
+            >
+              <li
+                v-for="(OpenPayment, index) in vendorMe?.OpenPayments"
+                :key="index"
+                class="flex w-full p-1 pt-2 relative"
+              >
+                <div class="flex-none grid grid-rows-1 place-content-start">
+                  <div class="pb-1">
+                    {{ $t('date') }}: {{ formatTime(OpenPayment.Timestamp) }},
```

Comment:
Maybe format the time like `04.12.2023 - 12:33` else it's to complicated to understand i think